src/main.lua

```diff
@@ -1,15 +1,12 @@
-local keyboard = require("keyboard/keyboard")
 local Board = require("board/board")
 local constants = require("constants")
 local Tetrion = require("tetrion/tetrion")
 
-local board = Board:new(10, 22)
-tetrion = Tetrion:new(board)
-
-keyboard.init(board)
 love.window.setMode(constants.WINDOW_WIDTH, constants.WINDOW_HEIGHT, nil)
 
-function love.draw()
-    board:render()
-    tetrion:render()
-end
+board = Board:new(10, 22)
+tetrion = Tetrion:new(board)
+
+require("love/draw")
+require("love/update")
+require("love/keypress")
```

Comment:
Hmm I don't know how to feel about this. I don't think we need a love/draw file just for the draw function. Preferably passing in the board and not making it a global would make more sense and if another board will (probably not) be added in the future. If you have any good reasons please elaborate :smile: or we can meet up :wink:

Reply:
For the love functions I wanted to group them all together. I can move them back to main.lua. 
I guess for the global board thing its because I was trying to get rid of the keyboard class, since everything will be done in the love function. let me play around with it some more

Reply:
For the love functions I wanted to group them all together. I can move them back to main.lua. 
I guess for the global board thing its because I was trying to get rid of the keyboard class, since everything will be done in the love function. let me play around with it some more